src/external-site/_vars-external-site.html

```diff
@@ -6,7 +6,7 @@
    ========================================================================== */
 
     // Find the external-urls in your local cfgov-refesh repo.
-    find . -name '*.html' ! -path './src/*' ! -path './node_modules/*' ! -path './_tests/*' \
+    find . -name '*.html' ! -path './*' ! -path '../node_modules/*' ! -path '../test/*' \
```

Comment:
Removed in https://github.com/cfpb/cfgov-refresh/commit/0721281b3f935582117a39e179c86fcd4dd61223